open_spiel/scripts/build_and_run_tests.sh

```diff
@@ -161,6 +161,11 @@ function print_skipping_tests {
   echo -e "\033[32m*** Skipping to run tests.\e[0m"
 }
 
+function execute_export_graph {
```

Comment:
Add to python tests, conditional on BUILD_WITH_TENSORFLOW_CC
https://github.com/deepmind/open_spiel/blob/master/open_spiel/python/CMakeLists.txt#L50

Reply:
Added.